### PR TITLE
docs: guide about where config

### LINKED ISSUE #38752: :where Selector 带来的兼容性问题（国内普及率只有61%）
### What problem does this feature solve?

在更新 5.0 版本的 Ant Design 后，新的 `css in js` 库生成了含有 `:where()` 的样式。

Screenshot from 2022-11-19 11-42-56

这导致所有不支持 `:where` 的浏览器（Chrome 81 才支持），都无法使用 Ant Design 的所有组件， 根据 [caniuse 提供的数据](https://caniuse.com/mdn-css_selectors_where)，:where 在国内的普及率只有61%，而它只是用于降低 css 优先级。

### What does the proposed API look like?

目前，postcss 没有提供[相关的 polyfill](https://github.com/csstools/postcss-plugins/discussions/509)，希望可以提供降级方案（至少不要让它完全不能用），或者避免使用这个 css 特性。

### PR DESCRIPTION
**Pull request by zombieJ**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link

resolve #38752

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |      Dynamic theme add how to replace `:where` selector for old browser.     |
| 🇨🇳 Chinese |    【要写到 changelog 里】 动态主题文档添加如何替换 `:where` 选择器以支持旧版游览器的指导。     |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Review by li-jia-nan — APPROVED** [review]

File: docs/react/customize-theme.zh-CN.md [hunk 1]
```diff
@@ -215,6 +215,21 @@ const theme = {
 };
 ```
 
+### 兼容性调整
+
+Ant Design 的 CSS-in-JS 默认通过 `:where` 选择器降低 CSS Selector 优先级。在某些场景下你如果需要支持的旧版游览器，你可以使用 `@ant-design/cssinjs` 取消默认的降权操作：
```
Comment (afc163) [inline docs/react/customize-theme.zh-CN.md:220]: 要写一下这样设置之后的副作用是啥

Comment (zombieJ) [reply docs/react/customize-theme.zh-CN.md:220]: > 要写一下这样设置之后的副作用是啥

Updated

Comment (afc163) [reply docs/react/customize-theme.zh-CN.md:220]: 这不是副作用吧，本质上要是说明为啥我们不『取消默认的降权操作』

Comment (afc163) [reply docs/react/customize-theme.zh-CN.md:220]: 就是说改了之后可能会遇到什么问题。

**Review by MadCcc — APPROVED** [review]

**Comment by dancerphil** [comment]
一个题外话，能不能把 layer 也做成类似的配置，用户可以开启，兼容性问题就自负。

**Comment by zombieJ** [comment]
> 一个题外话，能不能把 layer 也做成类似的配置，用户可以开启，兼容性问题就自负。

`@layer` 本身优先级过低，它有很多问题：

```css
@layer {
  .test {
    color: red;
  }

  .test:hover {
    color: blue;
  }
}

// hover 也会被这个覆盖
.test {
  color: green;
}
```

所以 v5 试了一下 layer 后就不再继续跟进了，用户除非全部都是 `@layer` 否则就用不了。

**Comment by dancerphil** [comment]
原来如此

**Pull request merged**